contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -220,14 +265,18 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
         ensureNonzeroAddress(tokenAddressIn);
         ensureNonzeroAddress(tokenAddressOut);
 
+        if (address(converterNetwork) == address(0)) {
+            revert InvalidConverterNetwork();
+        }
```

Comment:
I think this way we are making `converterNetwork` mandatory. I think we should be able to configure a token converter without converter network. So, I would revert the TX if the `converterNetwork` is zero, only if this is also true:

`conversionConfigurations[tokenAddressIn][tokenAddressOut].conversionAccess in (ConversionAccessibility.ONLY_FOR_CONVERTERS, ConversionAccessibility.ALL)`